code/game/objects/items/weapons/tools.dm

```diff
@@ -235,14 +235,17 @@
 
 /obj/item/weapon/weldingtool/attackby(obj/item/W as obj, mob/user as mob)
 	if(istype(W,/obj/item/weapon/screwdriver))
+		if (isrobot(loc))
+			user << span("alert", "You cannot modify your own welder!")
+			return
 		if(welding)
-			user << "<span class='danger'>Stop welding first!</span>"
+			user << span("danger", "Stop welding first!")

```

Comment:
There's really no point in doing this, except maybe a teeny tiny microscopic decrease in filesize 😛 

I wrote the span proc (now a define) as a shortcut to avoid having to write span class...etc. Its simply a convenience for the coder. Replacing one with the other has literally zero effect.